OneCryptoPkg/Include/Private/OneCryptoCrtLibSupport.h

```diff
@@ -0,0 +1,110 @@
+/** @file
+  One Crypto CRT Library that helps perform dependency injection 
+
+  Copyright (c) Microsoft Corporation.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef ONE_CRYPTO_CRT_LIB_SUPPORT_H_
+#define ONE_CRYPTO_CRT_LIB_SUPPORT_H_
```

Comment:
I'm not clear on the need for this file versus the other one. Going to see usage and come back.